src/utils/roomHelper.ts

```
@@ -15,6 +15,38 @@ export const checkUserInRoom = async (socket: Socket): Promise<{ room: string, r
+  try {
+    if (role === 'god') {
+      console.log("User is a god");
+      const room = await clientDB.collection('rooms').findOne({ code, 'gods.id': socketToCheck });
+
+      if (room && room.gods) {
+        return room.gods.find((g: any) => g.id === socketToCheck) || null;
+      }
+    } else if (role === 'player') {
+      const room = await clientDB.collection('rooms').findOne({ code, 'players.id': socketToCheck });
+
+      if (room && room.players) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)